src/controllers/audits.js

```diff
@@ -129,37 +146,51 @@ function AuditsController(dataAccess) {
       return badRequest('Audit type required');
     }
 
-    const { excludedURLs } = context.data;
+    const { excludedURLs, manualOverwrites } = context.data;
+    let hasUpdates = false;
 
+    const site = await dataAccess.getSiteByID(siteId);
+    if (!site) {
+      return notFound('Site not found');
+    }
+
+    const auditConfig = site.getAuditConfig();
+    const auditTypeConfig = auditConfig.getAuditTypeConfig(auditType);
+    if (!auditTypeConfig) {
+      return notFound('Audit type not found');
+    }
     if (Array.isArray(excludedURLs)) {
       for (const url of excludedURLs) {
         if (!isValidUrl(url)) {
           return badRequest('Invalid URL format');
         }
       }
-      // get audit type config
-      const site = await dataAccess.getSiteByID(siteId);
-      if (!site) {
-        return notFound('Site not found');
-      }
-      const auditConfig = site.getAuditConfig();
-      const auditTypeConfig = auditConfig.getAuditTypeConfig(auditType);
-      if (!auditTypeConfig) {
-        return notFound('Audit type not found');
-      }
+
+      hasUpdates = true;
 
       const newExcludedURLs = excludedURLs.length === 0
         ? []
         : Array.from(new Set([...(auditTypeConfig.getExcludedURLs() || []), ...excludedURLs]));
 
       auditTypeConfig.updateExcludedURLs(newExcludedURLs);
+    }
+
+    if (Array.isArray(manualOverwrites)) {
+      hasUpdates = true;
+
+      const existingOverrides = auditTypeConfig.getManualOverwrites();
+      const newManualOverwrites = manualOverwrites.length === 0 ? []
+        : mergeOverrides(existingOverrides, manualOverwrites);
```

Comment:
Since mergeOverrides will handle an empty array correctly, there is no need for the ternary operator checking manualOverwrites.length === 0. The function mergeOverrides can be used directly. 
```suggestion
  const newManualOverwrites = mergeOverrides(existingOverrides, manualOverwrites);
```

Reply:
here I was considering that when manualOverwrites is an [], to clear the existing overwrites, basically have "remove all overwrites", setting newManualOverwrites to []. Now, with this change when manualOverwrites is empty, it will simply return existingOverrides and not remove all overwrites.

Reply:
I would keep this option with removing all overwrites by sending [].
It might be helpful until we provided more granular APIs.